Dockerfile.autotune

```diff
@@ -16,7 +16,7 @@
 ##########################################################
 #            Build Docker Image
 ##########################################################
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9 as mvnbuild-jdk17
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10 as mvnbuild-jdk17
```

Comment:
Needs to updated in both places, BUILD and RUNTIME

Reply:
Updated, thank you